### PR TITLE
Find tenant from app_id and app_key

### PR DESCRIPTION
**Pull request by manu-d**
Expose new method to find tenant from key and secret

### CODE REVIEW

**Comment by BrunoChauvet** [comment]
@manu-d can you check the security issue?

File: lib/maestrano.rb [hunk 1]
```diff
@@ -127,6 +127,13 @@ def self.auto_configure(config_file_path = nil)
     raise "Error while fetching dynamic configuration: #{e}. Backtrace: #{e.backtrace}"
   end
 
+  def self.find_by_app_id_and_app_key(app_id, app_key)
```
Comment (BrunoChauvet) [inline lib/maestrano.rb]:
```ruby
Maestrano.configs.find { |_,v| v.param('api.id') == app_id && v.param('api.key') == app_key }&.first
```

**Comment by manu-d** [comment]
@BrunoChauvet  All good :)

**Comment by manu-d** [comment]
@BrunoChauvet 
All good now :)
(The ampersand was introduced in Ruby 2.3, and Travis tests also for Ruby 2.1 and 2.2)

**Review by BrunoChauvet — APPROVED** [review]

**Pull request merged**